public/locales/en/translation.json

```
@@ -10,6 +11,10 @@
         "customWine": "Select the Wine or Proton Binary",
         "default-install-path": "Choose Default Install Path",
         "error": {
+            "credentials": {
+                "message": "Your Crendentials have expired, Logout and Login Again!",
```

```suggestion
                "message": "Your credentials have expired, please logout and login again.",
```

Will need to be fixed in other files too.